faithcomesbyhearing/dbp-web
# FCBHDBP-441 Solomon Islands not listed as country on bible.is web

# Description
It has removed the Solomon Islands ( "SB" ) from territoryCode list to allow that Solomon Islands is displayed into country list. Also, it has been added to countryData.js file.

## Issue Link
https://fullstacklabs.atlassian.net/browse/FCBHDBP-441

## How Do I QA This
1. Navigate to [live.Bible.is](http://live.bible.is/)
2. Select the drop down menu where it says "English standard version / change Bible version” 
3. Tap country chip
4. Search for “Solomon islands”
5. The Solomon islands country should be rendered

## Screenshots
![Screenshot from 2022-09-12 16-40-00](https://user-images.githubusercontent.com/73488660/189763502-d646f4a9-e020-43ca-ac13-94746604feff.png)
